cmd/agent/servicestub.go

```diff
@@ -0,0 +1,10 @@
+// +build !windows
```

Comment:
IMO this should share a file prefix with `service_windows.go`, so I'd name this `service.go` instead of servicestub